scripts/openmc-ace-to-hdf5

```diff
@@ -111,25 +118,25 @@ for filename in ace_libraries:
             # Continuous-energy neutron data
```

Comment:
It looks like this was only really a problem for cross_sections.xml (I'll get that fixed). For xsdir and xsdata, the script was already checking for duplicate paths.

It would take quite a bit more work to actually be able to pick out specific tables rather than just specify files, so I'm going to leave this with the simple solution (removing duplicate entries).